src/app/showStartingPage.js

```diff
@@ -1,31 +1,34 @@
 export const showStartingPage = () => {
     const appScreen = document.querySelector('#pokequiz-app');
     const startingPageTemplate = 
-    `<div id="headerWithLogo">
-        <img src="https://static.wikia.nocookie.net/logo-timeline/images/8/8c/Pok%C3%A9mon.svg/revision/latest/scale-to-width-down/300?cb=20181024042907" alt='Pokemon' id='pokemonLogo'/>
+    `<div id="headerWithLogo" class="firstColumn spanInPortrait">
+        <img src="./static/assets/ui/logo.png" alt='Pokemon' id='pokemonLogo'/>
         <p id='quizLogo' class='fancyFontStyle'>Quiz</p>
     </div>
 
-        <ul id="selectModeMenu">
-            <li id='whoIsThatPokemonOption'>Who's that pokemon?</li>
-            <li id='whatItLooksLikeOption'>What it looks like?</li>
-            <li id='guessTheTypeOption'>Guess the type!</li>
+        <ul id="selectModeMenu" class="firstColumn spanInPortrait">
+            <li id='whoIsThatPokemonOption' class="buttonWithText">Who's that pokemon?</li>
+            <li id='whatItLooksLikeOption' class="buttonWithText">What it looks like?</li>
+            <li id='guessTheTypeOption' class="buttonWithText">Guess the type!</li>
         </ul>
 
-    <div id='enterNameAndPlayMenu'>
+    <div id='enterNameAndPlayMenu' class="secondColumn spanInPortrait">
         <p id='enterYourName' class='fancyFontStyle'>ENTER YOUR NAME</p>
         <div id='enterYourNameArea'>
             <img src='https://cdn4.iconfinder.com/data/icons/small-n-flat/24/user-alt-512.png' alt='playerIcon' id='playerIcon'/>
             <input type='text' id='enterYourNameInput'>
         </div>
-        <input type='button' id='startGameButton' value='PLAY'>
     </div>
 
-    <ul class='bottomOfThePageOptions'>
+    <div id='startGame' class="secondColumn spanInPortrait">
+        <input type='button' id='startGameButton' class="buttonWithText" value='PLAY'>
+    </div>
+
+    <ul class='bottomOfThePageOptions' class="firstColumn">
```

Comment:
small error: two 'class' attributes, it should be one with two classes :)
otherwise only one class is affected